src/Raven.Server/Documents/Replication/ResolveConflictOnReplicationConfigurationChange.cs

```diff
@@ -232,15 +232,15 @@ private void UpdateScriptResolvers()
                 var newChangeVector = _database.DocumentsStorage.GetNewChangeVector(context);
                 if (incoming.Doc != null)
                 {
-                    _database.DocumentsStorage.RevisionsStorage.Put(context, incoming.Id, incoming.Doc, incoming.Flags | DocumentFlags.Conflicted | DocumentFlags.HasRevisions,
+                    _database.DocumentsStorage.RevisionsStorage.Put(context, incoming.Id, incoming.Doc, incoming.Flags.Strip(DocumentFlags.FromClusterTransaction) | DocumentFlags.Conflicted | DocumentFlags.HasRevisions,
```

Comment:
Why are we removing important info (that the revision is from cluster tx) ?

Reply:
You are right. Was too eager about removing it..